src/hpr_routing.erl

```diff
@@ -71,6 +71,7 @@ handle_packet(Packet, StreamHandler) ->
 deliver_packet(_Packet, _StreamHandler, []) ->
     ok;
 deliver_packet(Packet, StreamHandler, [Route | Routes]) ->
+    hpr_packet_reporter:report_packet(Packet, Route),
```

Comment:
@michaeldjeffrey 
I can take care of the FIXME on L85 now, unless there is something more pressing for me to do.

Reply:
Go for it!!

Reply:
This has been done in
https://github.com/helium/helium-packet-router/pull/46